admiral/pkg/controller/common/types_test.go

```diff
@@ -191,3 +191,50 @@ func TestMapRange(t *testing.T) {
 	assert.Equal(t, 3, numOfIter)
 
 }
+
+func TestSidecarEgressGet(t *testing.T) {
+
+	egressMap := NewSidecarEgressMap()
+	egressMap.Put("pkey1", "pkey2", "fqdn", map[string]string{"pkey2": "pkey2"})
+
+	ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(3*time.Second))
+	defer cancel()
+
+	go func(ctx context.Context) {
```

Comment:
Adding a few comments in the relevant sections of the code will be useful.

Reply:
@nirvanagit , I wrote this test to reproduce the concurrent map access failure. Does it need a comment?

Reply:
Just adding comments on the producer and consumer side stating what is doing what, will enhance readability for everyone, even if they are not super familiar with channels.